migration-guide.md

```diff
@@ -0,0 +1,317 @@
+# Migration to Plotly 3.0.0
+There are many new and great features in Plotly 3.0 including deeper Jupyter integration, deeper figure validation, improved performance, and more. To get started right away with Plotly, check out the [tutorial](#getting-started) below.
+
+## 3.0.0rc9 [08-06-2018]
+### Added
+- Jupyter integration - run `help(go.FigureWidget)` for more information
+- update traces interactively
+- Traces can be added and updated interactively by simply assigning to properties
+- The full Traces and Layout API is generated from the plotly schema to provide a great experience for interactive use in the notebook
+- Support for setting array properties as numpy arrays. When numpy arrays are used, ipywidgets binary serialization protocol is used to avoid converting these to JSON strings.
+- Context manager API for animation. Run `help(go.Figure().batch_animate)` for the full doc string.
+- Programmatic export of figures to static SVG images (and PNG and PDF with cairosvg installed).
```

Comment:
I removed this static image export approach now that we're going to integrate Orca soon.